### DISCUSSION TITLE
Adding a material like timepicker

### DISCUSSION THREAD

**Discussion by Ghoral** [shadcn-ui/ui#584] [Ideas]
Hi,
Thanks for this amazing library.

Currently, our project lacks a modern and visually appealing time picker component. It would be better if there is a timepicker like this is available.

<img width="317" alt="Screenshot 2023-06-14 at 13 50 20" src="https://github.com/shadcn/ui/assets/49188349/9a2f7d99-f520-4d96-9d55-6944849f6bdb">

**Comment by michaeldesbruslais** [comment]
![time](https://github.com/shadcn/ui/assets/7108758/032079e2-0612-462c-8ab7-692fdccb02be)
potentially something like this may also be another way to approach this

**Comment by Schmell** [comment]
Yes please. This would be super nice and could be rolled into datetime easily

**Reply by Schmell** [reply]
i love the clock style one

**Comment by eximius313** [comment]
I love this idea! It's so much easier to select time using this component!
